sandbox/rocky/tf/algos/ddpg/ddpg.py

```diff
@@ -0,0 +1,215 @@
+from sandbox.rocky.tf.algos.ddpg.ddpg_base_net import ActorNet, CriticNet
+from sandbox.rocky.tf.algos.ddpg.replay_buffer import ReplayBuffer
+
+import tensorflow as tf
+from copy import copy
+import numpy as np
+import os
+
+
+class DDPG(object):
+    def __init__(self,
+                 env,
+                 gamma=0.99,
+                 tau=0.01,
+                 observation_range=(-5, 5),
+                 action_range=(-1, 1),
+                 actor_lr=1e-4,
+                 critic_lr=1e-3,
+                 reward_scale=1,
```

Comment:
Can you leave detailed comments on what each of these parameters does? Even better if you can reference them back to the constants in the equations in a paper.

Reply:
Done